amoro-ams/src/main/java/org/apache/amoro/server/AmoroManagementConf.java

```diff
@@ -52,6 +52,13 @@ public class AmoroManagementConf {
           .defaultValue("admin")
           .withDescription("The administrator password");
 
+  /** Enable master & slave mode, which supports horizontal scaling of AMS. */
+  public static final ConfigOption<Boolean> USE_MASTER_SLAVE_MODE =
```

Comment:
I’m wondering whether we could control the single-node and multi-node modes with a configuration option, with the default being the single-node mode.